src/utils/dataMap.ts

```diff
@@ -4,6 +4,12 @@ import { formatCategoryToGameNameKR } from './gameFormatter';
 
 const DELIMITER = '&iquest';
 
+export const joinTitleByDelimiter = (array: string[]) => {
+  const joinedTitle = array.join(DELIMITER);
+
+  return joinedTitle;
```

Comment:
```suggestion
export const joinTitleByDelimiter = (array: string[]) => {
  return array.join(DELIMITER); 
```

이렇게 적어도 괜찮지 않을까요?

Reply:
```suggestion
export const joinTitleByDelimiter = (array: string[]) => array.join(DELIMITER);
```

추가로, 한줄짜리니깐 이렇게 중괄호랑 리턴까지 생략하고 적으면 좋을거같습니다!